codegenerator/cli/templates/static/codegen/src/globalState/GlobalState.res

```diff
@@ -786,337 +779,332 @@ let checkAndFetchForChain = (
   //required args
   ~state,
   ~dispatchAction,
-) =>
-  async chain => {
-    let chainFetcher = state.chainManager.chainFetchers->ChainMap.get(chain)
-    let {chainConfig: {chainWorker}, logger, currentBlockHeight} = chainFetcher
-
-    if !isRollingBack(state) {
-      let (nextQuery, nextStateIfChangeRequired) =
-        chainFetcher->ChainFetcher.getNextQuery(~maxPerChainQueueSize=state.maxPerChainQueueSize)
-
-      switch nextStateIfChangeRequired {
```

Comment:
Most of these changes are just formatting. Either review with whitespace disabled or I separated the commit with the formatting changes so you should be able to see exactly what changed.